region/maridia/outer/Glass Tunnel.json

```diff
@@ -586,6 +586,38 @@
         "Ride up the right wall just above the doorway, then turnaround before and after hitting the platform below the tube."
       ]
     },
+    {
+      "link": [2, 4],
+      "name": "Grapple Teleport into Grapple Jump (or Tube Intact)",
+      "entranceCondition": {
+        "comeInWithGrappleTeleport": {
+          "blockPositions": [[5, 3]]
+        }
+      },
+      "requires": [
+        {"doorUnlockedAtNode": 4},
+        {"or": [
+          {"and": [
+            "canGrappleJump",
```

Comment:
Couldn't do it.
It's a harder grapple than the fish tank one because of the platform under the door.
You have to retract the grapple a bit before you hit it.